roles/openshift_common/vars/main.yml

```diff
@@ -6,4 +6,4 @@
 # interfaces)
 os_firewall_use_firewalld: False
 
-openshift_data_dir: /var/lib/openshift
+openshift_data_dir: /var/lib/{{ product_type }}
```

Comment:
They are. This was referenced by @sdodson yesterday as something that changed after this work was done (and would fixup post spec file updates). However, we can easily change it now so I'll go ahead and do it :smile: